pkg/streamingpromql/testdata/ours/native_histograms.test

```diff
@@ -145,3 +144,32 @@ load 1m
 eval range from 0 to 4m step 1m sum by (env) (rate(incr_histogram[5m]))
 	{env="a"}	_ {{count:0.01 sum:0.02 offset:1 buckets:[0.01]}} {{count:0.016666666666666666 sum:0.03333333333333333 offset:1 buckets:[0.016666666666666666]}} {{count:0.023333333333333334 sum:0.04666666666666667 offset:1 buckets:[0.023333333333333334]}} {{count:0.03333333333333333 sum:0.06666666666666667 offset:1 buckets:[0.03333333333333333]}}
 	{env="b"}	_ {{count:0.01 sum:0.02 offset:1 buckets:[0.01]}} {{count:0.016666666666666666 sum:0.03333333333333333 offset:1 buckets:[0.016666666666666666]}} {{count:0.023333333333333334 sum:0.04666666666666667 offset:1 buckets:[0.023333333333333334]}} {{count:0.03333333333333333 sum:0.06666666666666667 offset:1 buckets:[0.03333333333333333]}}
+
+clear
+
+# Test mixing exponential and custom buckets.
+load 6m
+  metric{series="exponential"}         {{sum:4 count:3 buckets:[1 2 1]}}  _                                                                 {{sum:4 count:3 buckets:[1 2 1]}}
+  metric{series="other-exponential"}   {{sum:3 count:2 buckets:[1 1 1]}}  _                                                                 {{sum:3 count:2 buckets:[1 1 1]}}
+  metric{series="custom"}              _                                  {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}}     {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}}
+  metric{series="other-custom"}        _                                  {{schema:-53 sum:15 count:2 custom_values:[5 10] buckets:[0 2]}}  {{schema:-53 sum:15 count:2 custom_values:[5 10] buckets:[0 2]}}
+
+# T=0: only exponential
+# T=6: only custom
+# T=12: mixed, should be ignored and emit an warning
```

Comment:
would be good to see some valid points after an ignored point due to the way point count is decremented.

Reply:
Done: 20a3fc0